.circleci/config.yml

```diff
@@ -31,13 +30,10 @@ jobs:
           # fallback to using the latest cache if no exact match is found
           - v1-dependencies-
 
-      - run: mvn dependency:go-offline
```

Comment:
Doesn't this mean that we won't be caching dependencies?

Reply:
No idea. First we should get the tests passing. We can optimize later.